website/Contribution/CONTRIBUTING.rst

```diff
@@ -94,6 +94,8 @@ A module is organized in a few directories:
 * `examples/`: external files
   `lib/`, ...
 * `models/`: model definitions
+* `readme/`: contains rst files required to make README file of module, for example `CONFIGURE.rst`, `CONTRIBUTORS.rst`...
+  use `<https://github.com/OCA/maintainer-tools/tree/master/template/module/README.rst>`_ to generate README.rst file.
```

Comment:
The link to what to use seems incorrect. Shouldn't you link to `oca-gen-addons-readme`?